src/main/java/org/pih/petl/job/type/SqlServerImportJob.java

```diff
@@ -65,13 +65,13 @@ public void execute(final ExecutionContext context) throws Exception {
         String sourceContextFileName = config.getString("extract", "context");
         if (sourceContextFileName != null) {
             ConfigFile sourceContextFile = appConfig.getConfigFile(sourceContextFileName);
-            sourceContextStatements = sourceContextFile.getContents();
+            sourceContextStatements = sourceContextFile.getContentsWithVariableReplacement(config.getVariables());
         }
 
         // Get source query
         String sourceQueryFileName = config.getString("extract", "query");
         ConfigFile sourceQueryFile = appConfig.getConfigFile(sourceQueryFileName);
-        String sourceQuery = sourceContextStatements + sourceQueryFile.getContents();
+        String sourceQuery = sourceContextStatements + sourceQueryFile.getContentsWithVariableReplacement(config.getVariables());
```

Comment:
Adding this in allows us to support variable replacement within the source query sql file.